lib/pages/login-page.js

```diff
@@ -22,28 +22,23 @@ export default class LoginPage extends AsyncBaseContainer {
 		const passwordSelector = By.css( '#password' );
 
 		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
-		await this.driver.sleep( 1000 );
+		await driverHelper.clickWhenFocused( driver, userNameSelector );
 		await driverHelper.setWhenSettable( driver, userNameSelector, username );
-		await this.driver.sleep( 1000 );
 		await driver.findElement( userNameSelector ).sendKeys( Key.ENTER );
 
-		await this.driver.sleep( 1000 );
-
 		if ( emailSSO === false ) {
+			await driverHelper.waitTillPresentAndDisplayed( driver, passwordSelector );
+			await driverHelper.clickWhenFocused( driver, passwordSelector );
```

Comment:
Implementing `waitTillPresentAndDisplayed` and a new `driverHelper` method `clickWhenFocused` to address the sleep statement that was previously here.